integration_tests/run_pyspark_from_build.sh

```diff
@@ -93,6 +93,12 @@ else
     RUN_DIR="$SCRIPTPATH"/target/run_dir
     mkdir -p "$RUN_DIR"
     cd "$RUN_DIR"
+
+    ## Under cloud environment, overwrite the '--rootdir' param to point to the working directory of each excutor
+    LOCAL_ROOTDIR=${LOCAL_ROOTDIR:-"$SCRIPTPATH"}
+    ## Under cloud environment, overwrite the '--std_input_path' param to point to the distributed file path
+    CLOUD_INPUTPATH=${CLOUD_INPUTPATH:-"$SCRIPTPATH"}
```

Comment:
nit: this is not a **CLOUD**_INPUTPATH in all cases. I think it would be better to just call it `INPUT_PATH` i.e.

```
INPUTPATH=${INPUTPATH:-"$SCRIPTPATH"}
```

Reply:
Sounds good, updated!